jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/handler/FromConfigRegistryAuthHandlerTest.java

```diff
@@ -39,8 +44,8 @@
     @Mocked
     private KitLogger log;
 
-    @Rule
-    public ExpectedException expectedException = ExpectedException.none();
+    /*@Rule
+    public ExpectedException expectedException = ExpectedException.none();*/      //removed expectedException @Rule
```

Comment:
Instead of commenting them out, please remove it.

Reply:
removed